src/prefect/environments/storage/docker.py

```diff
@@ -453,19 +462,30 @@ def create_dockerfile_object(self, directory: str) -> str:
         Returns:
             - str: the absolute file path to the Dockerfile
         """
+        # Either load the base commands from the specified dockerfile or define a FROM
+        if self.dockerfile:
+            with open(self.dockerfile, "r") as contents:
+                base_commands = contents.read()
+        else:
+            base_commands = "FROM {base_image}".format(base_image=self.base_image)
+
```

Comment:
Moved from the bottom of the variable setup here because it's the first templated item

`contents.read()` is no longer indented using `textwrap.indent` here because the number of spaces to indent by is dependent on the indentation of the code further down.